pages/docs/adapters/su2/adapter-su2-overiew.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Overview
+permalink: adapter-su2-overview.html
+keywords: adapter, su2, development, modules
+summary: ""
```

Comment:
This appears on top of the overview page and we want every page to have a summary.